src/lib/project.ts

```diff
@@ -56,32 +43,22 @@ export interface ProjectInfo {
 
 export interface Tree {
   path: string;
-  info: EntryInfo; // TODO: Remove this after we have migrated to Heartwood.
   entries: Array<Entry>;
   stats: Stats;
   name: string;
   kind: Kind;
   lastCommit: CommitHeader;
 }
 
-// TODO: Remove "TREE" and "BLOB" after we have migrated to Heartwood.
-type Kind = "tree" | "blob" | "TREE" | "BLOB";
+type Kind = "tree" | "blob";
 
 export interface Stats {
   commits: number;
   contributors: number;
 }
 
-// TODO: Remove this after we have migrated to Heartwood.
-export interface EntryInfo {
-  name: string;
-  objectType: Kind;
-  lastCommit: CommitHeader;
-}
-
 export interface Entry {
   path: string;
-  info: EntryInfo; // TODO: Remove this after we have migrated to Heartwood.
   name: string;
   kind: Kind;
   lastCommit: CommitHeader;
```

Comment:
We should be able to remove the `lastCommit` property in the `Entry` or in the `Blob` interface

See: https://github.com/radicle-dev/heartwood/blob/9d244b221a167c1c2bacbd00ecefc1a44ecab552/radicle-httpd/src/api/json.rs#L68

Reply:
This is why we need zod, we'd get an exception if there was a discrepancy like this. :P
`Blob` seems to have a `lastCommit` if I'm not misreading the code? Will remove from `Entry`.

f99248fc07a7ab02411565b63793ca66a7956c71

Reply:
Fully agree on having zod :100: